R/make_states.R

```diff
@@ -0,0 +1,39 @@
+make_state_map_one <- function(state, geometry = TRUE, epsg = alarm_epsg(state)) {
+
+    nd <- geomander::get_alarm(state = state, geometry = geometry, epsg = epsg)
+
+    nd <- nd %>%
+        summarise(dplyr::across(tidyselect::starts_with("pop"), sum),
+                  dplyr::across(tidyselect::starts_with("vap"), sum),
+                  dplyr::across(tidyselect::matches("_(dem|rep)_"), sum),
+                  dplyr::across(tidyselect::matches("^a[dr]v_"), sum),
+                  geometry = sf::st_union(geometry))
+
+    id_nrv <- stringr::str_detect(colnames(nd), "arv_")
+    id_ndv <- stringr::str_detect(colnames(nd), "adv_")
+
+    nd <- nd %>%
+        mutate(nrv = rowMeans(as.data.frame(nd)[,id_nrv]),
+               ndv = rowMeans(as.data.frame(nd)[,id_ndv]),
+               nrv = round(nrv, 1),
+               ndv = round(ndv, 1)) %>%
+        relocate(geometry, .after = ndv)
+
+    redist::redist_map(nd, pop_col = pop, ndists = 1, pop_tol = 0.01) %>%
```

Comment:
I think you should do `pop_call="pop"` which will avoid an R CHECK warning. That may not work though.

Can you also ditch the pipe and just surround this line with `suppressWarnings()` directly?

Reply:
Think @mzhao80 [fixed these](https://github.com/alarm-redist/alarmdata/pull/6/commits/0d7ee387880cce445db0a368b5777d3e8c6231f7) -- thanks!

Reply:
Done [here](https://github.com/alarm-redist/alarmdata/pull/6/commits/0d4ba1978be3d611b7cd55be7c0efcbdef8443a2#diff-a9ed73b051d4d4e2fb05869189ffab044dca4277b76001b12883011d4e36a5d8R22).

Reply:
Should we do `"pop"` or `.data$pop`?

Reply:
I think `.data` is specific to `dplyr` functions, so if `"pop"` works let's do that